README.md

```diff
@@ -76,10 +76,12 @@ An example of a basic metadata schema:
     - **type_category:** These group different sets of `type` properties into a single superset. These are: `integer`, `float`, `string`, `timestamp`, `bool`, `list`, `struct`. For example we class `int8, int16, int32, int64, uint8, uint16, uint32, uint64` as `integer`. It allows users to give more generic types if your data is not coming from a system or output with strict types (i.e. data exported from Excel or an unknown origin). The Metadata class has default type values for each given `type_category`. See the `default_type_category_lookup` attribute of the `Metadata` class to see said defaults. This field is required if `type` is not set.
     - **description:** Description of the column.
     - **enum:** List of what values that column can take. _(Same as the standardised json schema keyword)._
-    - **pattern**: Regex pattern that value has to to match (for string type_categories only). _(Same as the standardised json schema keyword)._
-    - **minLength / maxLength**: The minimum and maximum length of the string (for string type_categories only). _(Same as the standardised json schema keyword)._
-    - **minimum / maximum**: The minumum and maximum value a numerical type can take (for integer and float type_categories only).
-- **partitions**: List of what columns in your dataset are partitions.
+    - **pattern:** Regex pattern that value has to to match (for string type_categories only). _(Same as the standardised json schema keyword)._
+    - **minLength / maxLength:** The minimum and maximum length of the string (for string type_categories only). _(Same as the standardised json schema keyword)._
+    - **minimum / maximum:** The minumum and maximum value a numerical type can take (for integer and float type_categories only).
+- **partitions:** List of what columns in your dataset are partitions.
+- **table_location:** the location in of the table. This is optional and primarily for overlaying schemas.
+- **database_name:**  the name of the database this table belongs to. Optional as well for the same reason as table_location.
```

Comment:
```suggestion
- **database_name:**  the name of the database this table belongs to (optional).
```
I think this is sufficient? Atm `table_location` and `database_name` are used for the same converter (Glue) but I can that these wouldn't be so tightly coupled e.g. if you just want to read data in, why not use the `table_location` property to do so - this would not require a `database_name`. So I think it is better to not couple the properties in the docs. The docs on glue converter can state what it uses properties for.